doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -232,6 +233,14 @@ Example of regular TLS logging:
       "issuerdn": "C=US, O=Google Inc, CN=Google Internet Authority G2"
   }
 
+Example of regular TLS logging for resumed sessions:
+
+::
+
+  "tls": {
+      "session": "resumed"
```

Comment:
can we think of other values that "session" could get? If not then I think we should turn it into a bool.